style/redux/README.md

```diff
@@ -0,0 +1,195 @@
+# Redux Style Guide
+
+## Table of Contents
+
+- [Redux directory](#redux-directory)
+- [Action creators](#action-creators)
+- [Action types](#action-types)
+- [Reducers](#reducers)
+- [Connecting React components](#connecting-react-components)
+
+## Redux directory
+
+Redux related logic should all live under a separate directory `redux/` in
+order to isolate it from app logic
+
+## Action creators
+
+- Group related action creators in one file.
+- Use [redux-promise-middleware] for asynchronous actions, and/or [redux-thunk]
+  if you need more flexibility.
+- Expose each action creator as a named export.
+- Don't use a default export.
+
+```js
+// actions/todo.js
+import { todo } from '../constants/actionTypes';
+import * as api from '../../utils/api';
+
+export function addTodoSync() {
+  return {
+    type: todo.addTodoSync,
+    payload: { foo: 'bar' },
+  };
+}
+
+export function fetchTodoAsync() {
+  return {
+    type: todo.fetchTodo,
+    payload: api.fetchTodo(),
+  };
+}
+```
+
+Action creators are functions which return an action object. An action object
+contains a `type` and optionally a `payload`, `error` and `meta`data. Action
+objects should follow the [Flux Standard Action] schema.
+
+We could also use the `createAction` helper of [redux-actions], but other than
+enforcing FSA it doesn't do much here in terms of reducing boilerplate. In fact
+`createAction` is less readable and doesn't let us easily export named
+functions. Note that only `type` is mandatory.
+
+So far the common practice is to group files by type rather than by feature, and
+this style guide follows that pattern. But there's an alternative pattern -
+[ducks] - a proposal to bundle action creators with their reducers into a single
+file. Such a bundle is called a 'duck'. Its goal is to move towards a modular
+application structure.
+
+[redux-promise-middleware]: https://github.com/pburtchaell/redux-promise-middleware
+[redux-thunk]: https://github.com/gaearon/redux-thunk
+[Flux Standard Action]: https://github.com/acdlite/flux-standard-action
+[redux-actions]: https://github.com/acdlite/redux-actions
+[ducks]: https://github.com/erikras/ducks-modular-redux
+
+## Action types
+
+- Put action types in one file of constants.
+- Group related action types in an object.
+- Expose each object as a named export.
+- Expose all action types as the default export.
+
+```js
+// constants/actionTypes.js
+export const todo = {
+  fetch: 'FETCH_TODO',
+  set: 'SET_TODO',
+};
+
+export default {
+  todo,
+};
+```
+
+Action types identify the nature of the action that has occurred. Two actions
+with the same type MUST be strictly equivalent (using ===). By convention, type
+is usually a string constant or a Symbol.
+
+## Reducers
+
+- Expose reducers as the default export.
+- Expose handlers as named exports for unit testing.
+- Use constants instead of inline strings for action types.
+- Always define an `INITIAL_STATE` variable.
```

Comment:
Ok would a simple change from `Always define` to `If applicable, define` suffice?